src/pyodide/python-entrypoint-helper.ts

```diff
@@ -217,7 +217,7 @@ async function initPyInstance(
       `There is no '${className}' class defined in the Python Worker's main module`
     );
   }
-  const res = await doPyCall(pyClassConstructor, args);
```

Comment:
I think this was required for some reason, @hoodmane I think did this.

Reply:
Yeah we need the `await` here since in 0.27 it will return a promise.

Reply:
It looked to me like we are awaiting the result of a class constructor here, so it returning a promise didn't make sense to me. What did I miss?

Reply:
JSPI makes all calls into webassembly code return promises. The class constructor can stack switch, which looks synchronous from inside of webassembly but looks async from JS.